### PR TITLE
elixir: add deprecated Bitwise rules

### PR DESCRIPTION
**Pull request by wingyplus**
Submit 3 `Bitwise` module rules:

1. Rewrite `Bitwise.^^^` into `Bitwise.bxor/2`.
2. Rewrite `Bitwise.~~~` into `Bitwise.bnot/1`.
3. Rewrite `use Bitwise` into `import Bitwise`.

### CODE REVIEW

File: elixir/lang/best-practice/deprecated-bnot-operator.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+rules:
+  - id: deprecated_bnot_operator
+    message: >-
+      The bitwise operator (`^^^`) is already deprecated. Recommended to use `Bitwise.bnot/2` instead.
```
Comment (colleend) [inline elixir/lang/best-practice/deprecated-bnot-operator.yaml]:
```suggestion
      The bitwise operator (`^^^`) is already deprecated. Please use use `Bitwise.bnot($VAL)` instead.
```

File: elixir/lang/best-practice/deprecated-bxor-operator.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+rules:
+  - id: deprecated_bxor_operator
+    message: >-
+      The bitwise operator (`^^^`) is already deprecated. Recommended to use `Bitwise.bxor/2` instead.
```
Comment (colleend) [inline elixir/lang/best-practice/deprecated-bxor-operator.yaml]:
```suggestion
      The bitwise operator (`^^^`) is already deprecated. Please use `Bitwise.bxor($LEFT, $RIGHT)` instead.
```

File: elixir/lang/best-practice/deprecated-use-bitwise.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+rules:
+  - id: deprecated_use_bitwise
+    message: >-
+      The `use Bitwise` is already deprecated. Recommended to use `import Bitwise` instead.
```
Comment (colleend) [inline elixir/lang/best-practice/deprecated-use-bitwise.yaml]:
```suggestion
      The syntax `use Bitwise` is already deprecated. Please use `import Bitwise` instead.
```

**Review by colleend — CHANGES_REQUESTED** [review]
Made a few small changes to the comments, but otherwise looks good to me. Thanks for submitting these!

File: elixir/lang/best-practice/deprecated-bnot-operator.yaml (hunk 1 above)
Comment (wingyplus) [inline elixir/lang/best-practice/deprecated-bnot-operator.yaml]:
```suggestion
      The bitwise operator (`^^^`) is already deprecated. Please use `Bitwise.bnot($VAL)` instead.
```

Comment (wingyplus) [reply elixir/lang/best-practice/deprecated-bnot-operator.yaml]: This already fixed in https://github.com/returntocorp/semgrep-rules/pull/3064/commits/4e9afa94685baeac65246f924fe5280986208b0a

**Comment by wingyplus** [comment]
@colleend done. :)

**Review by colleend — APPROVED** [review]
lgtm!

**Pull request merged**